### DISCUSSION TITLE
Alist挂载手机本地存储后访问会报错中断alist服务

### DISCUSSION THREAD

**Discussion by wolffy1998** [AlistGo/alist#5887] [Q&A]
环境：荣耀9i（安卓9）基于Termux安装Alist，清华镜像源，Termux已授权访问本机文件。

**问题描述：**
    Termux安装Alist后，挂载手机本机目录后（Termux和手机存储目录都试过），通过http请求响应的链接后，Termux终端会报错，然后alist服务中断。

**Termux终端报错信息：**

goroutine 74 [syscall]:
syscall.Syscall6(0x4000a24c08?, 0x63fcfe74fc?, 0x400029e4c0?, 0x8?, 0x4000a24c00?, 0x400029e4c0?, 0x8?)
        /home/builder/.termux-build/_cache/go1.21.5/src/syscall/syscall_linux.go:91 +0x2c fp=0x4000a24bc0 sp=0x4000a24b30 pc=0x63fcd4c41c
syscall.Syscall6(0x123, 0xffffffffffffff9c, 0x400029e4f0, 0x0, 0x8e0, 0x4000aaa200, 0x0)
        <autogenerated>:1 +0x2c fp=0x4000a24c10 sp=0x4000a24bc0 pc=0x63fcd4d07c
golang.org/x/sys/unix.Statx(0x4000d06338?, {0x400029e4c0?, 0x4000a24d18?}, 0x63fcd83b10?, 0x4000a24d18?, 0x4000aaa200)
        /home/builder/go/pkg/mod/golang.org/x/sys@v0.15.0/unix/zsyscall_linux.go:1593 +0xa8 fp=0x4000a24cb0 sp=0x4000a24c10 pc=0x63fcfe7568
github.com/djherbis/times.statX({0x400029e4c0, 0x8})
        /home/builder/go/pkg/mod/github.com/djherbis/times@v1.6.0/times_linux.go:76 +0x5c fp=0x4000a24d00 sp=0x4000a24cb0 pc=0x63fdc338fc
github.com/djherbis/times.Stat({0x400029e4c0, 0x8})
        /home/builder/go/pkg/mod/github.com/djherbis/times@v1.6.0/times_linux.go:61 +0x3c fp=0x4000a24d40 sp=0x4000a24d00 pc=0x63fdc337ec
github.com/alist-org/alist/v3/drivers/local.(*Local).Get(0x63fb821fed?, {0x63fe420950?, 0x4000164000?}, {0x63fb821fed?, 0x4000a24fd8?})
        /home/builder/.termux-build/alist/src/drivers/local/driver.go:160 +0x124 fp=0x4000a24e90 sp=0x4000a24d40 pc=0x63fdc74c14
github.com/alist-org/alist/v3/internal/op.Get({0x63fe4175e8, 0x4000d04100}, {0x63fe420950?, 0x4000164000?}, {0x63fb821fed?, 0x63fd147308?})
        /home/builder/.termux-build/alist/src/internal/op/fs.go:171 +0xc4 fp=0x4000a24ff0 sp=0x4000a24e90 pc=0x63fd147f94
github.com/alist-org/alist/v3/internal/op.GetUnwrap({0x63fe4175e8?, 0x4000d04100?}, {0x63fe420950?, 0x4000164000?}, {0x63fb821fed?, 0x0?})
        /home/builder/.termux-build/alist/src/internal/op/fs.go:233 +0x30 fp=0x4000a25050 sp=0x4000a24ff0 pc=0x63fd148670
github.com/alist-org/alist/v3/internal/op.List({0x63fe4175e8?, 0x4000d04100}, {0x63fe420950, 0x4000164000}, {0x63fb821fed, 0x1}, {{0x400029e368?, 0x1?}, 0xe8?}, {0x4000a252e7, ...})
        /home/builder/.termux-build/alist/src/internal/op/fs.go:116 +0x1d8 fp=0x4000a25270 sp=0x4000a25050 pc=0x63fd147328
github.com/alist-org/alist/v3/internal/fs.list({0x63fe4175e8, 0x4000d04100}, {0x400029e368, 0x6}, 0x4000a25456)
        /home/builder/.termux-build/alist/src/internal/fs/list.go:25 +0x1a4 fp=0x4000a25380 sp=0x4000a25270 pc=0x63fd5969c4
github.com/alist-org/alist/v3/internal/fs.List({0x63fe4175e8?, 0x4000d04100?}, {0x400029e368, 0x6}, 0x4000a25456)
        /home/builder/.termux-build/alist/src/internal/fs/fs.go:22 +0x30 fp=0x4000a25400 sp=0x4000a25380 pc=0x63fd595590
github.com/alist-org/alist/v3/server/handles.FsList(0x4000d04100?)
        /home/builder/.termux-build/alist/src/server/handles/fsread.go:86 +0x278 fp=0x4000a25570 sp=0x4000a25400 pc=0x63fddd8f98
github.com/gin-gonic/gin.(*Context).Next(...)
        /home/builder/go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174
github.com/alist-org/alist/v3/server/middlewares.Auth(0x4000d04100)
        /home/builder/.termux-build/alist/src/server/middlewares/auth.go:73 +0x4bc fp=0x4000a25680 sp=0x4000a25570 pc=0x63fdd9633c
github.com/gin-gonic/gin.(*Context).Next(...)
        /home/builder/go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174
github.com/alist-org/alist/v3/server/middlewares.StoragesLoaded(0x4000d04100)
        /home/builder/.termux-build/alist/src/server/middlewares/check.go:14 +0xa8 fp=0x4000a25760 sp=0x4000a25680 pc=0x63fdd96b88
github.com/gin-gonic/gin.(*Context).Next(...)
        /home/builder/go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174
github.com/gin-gonic/gin.CustomRecoveryWithWriter.func1(0x4000d04100)
        /home/builder/go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/recovery.go:102 +0x80 fp=0x4000a257b0 sp=0x4000a25760 pc=0x63fd588450
github.com/gin-gonic/gin.(*Context).Next(...)
        /home/builder/go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174
github.com/gin-gonic/gin.LoggerWithConfig.func1(0x4000d04100)
        /home/builder/go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/logger.go:240 +0xb0 fp=0x4000a25960 sp=0x4000a257b0 pc=0x63fd5877f0
github.com/gin-gonic/gin.(*Context).Next(...)
        /home/builder/go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174
github.com/gin-gonic/gin.(*Engine).handleHTTPRequest(0x40001b1860, 0x4000d04100)
        /home/builder/go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/gin.go:620 +0x524 fp=0x4000a25af0 sp=0x4000a25960 pc=0x63fd586924
github.com/gin-

**Comment by DOOI123** [comment]
出现同样报错

**Comment by njnu1115** [comment]
https://github.com/alist-org/alist/discussions/6233
